addon/services/panel-actions.js

```diff
@@ -21,29 +21,35 @@ const State = EmberObject.extend({
   group: null
 });
 
-export default Service.extend({
-  _registry: EmberObject.create({
-    keys: A([]),
+const Registry = EmberObject.extend({
+  keys: computed(function() {
+    return A([]);
+  }),
 
-    unknownProperty: function(name) {
-      const state = State.create();
+  // probably not too safe, should only be used in tests
+  reset() {
+    this.get('keys')
+      .map(i => i) // copy, so we dont mess with binding/loops
+      .forEach((key) => {
+        delete this[key];
+      });
 
-      this.get('keys').addObject(name);
-      this.set(name, state);
+    this.get('keys').clear();
+  },
+});
 
-      return state;
-    },
+export default Service.extend({
+  _registry: computed(function() {
+    return Registry.create({
+      unknownProperty: function(name) {
```

Comment:
Why do we define the `unknownProperty` here in the `.create` vs up above in the `.extend`?

Reply:
I was getting a max callstack error in the `ember-default` testing scenario when defining the `unknownProperty` in the extend - https://travis-ci.org/adopted-ember-addons/ember-collapsible-panel/builds/626881977.

Reply:
Somehow the `unknownProperty` function gets `setUnknownProperty` as the first argument (= key).